pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -0,0 +1,78 @@
+import json
+import sys
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import transaction
+
+from account import TGAS
+from common.base import Transaction
+
+
+class TransferFT(Transaction):
+
+    def __init__(self, ft, sender, recipient_id, how_much=1, tgas=300):
+        super().__init__()
+        self.ft = ft
+        self.sender = sender
+        self.recipient_id = recipient_id
+        self.how_much = how_much
+        self.attached_tgas = tgas
```

Comment:
mega nit: if the constructors of `TransferFT` never set the `tgas` arg, could just hardcode it here. personal preference though I guess. Not a big deal